src/main/java/CodeCheck/Main.java

```diff
@@ -16,13 +16,24 @@ public static void main(String[] args) throws Exception {
     private static void runModel() throws Exception {
         File baseCodeDir = new File(Util.PATH_TO_CODE);
 
-        if (!baseCodeDir.isDirectory()) baseCodeDir.mkdir();
```

Comment:
It's going to crash if the `baseCodeDir` doesn't exist, which is why I added this piece of code. If the directory doesn't exist, then create it, otherwise do nothing.